test/run_test.py

```diff
@@ -66,8 +66,7 @@ def skip_test_p(name: str) -> bool:
             rc |= name in blocklisted_tests
         return rc
     cwd = pathlib.Path(__file__).resolve().parent if base_dir is None else base_dir
-    # This supports symlinks, so we can link domain library tests like functorch
-    # to PyTorch test directory
+    # This supports symlinks, so we can link domain library tests to PyTorch test directory
     all_py_files = [pathlib.Path(p) for p in glob.glob(f"{cwd}/**/test_*.py", recursive=True)]
```

Comment:
cc @huydhn who did this change originally to get us sharding

Just want to check if you think we should keep it like this to support future symlinks or change back anything else originally done in #85321

Reply:
I would prefer to keep the symlink logic here because it's more flexible than the old logic in itself.  For functorch, this PR has already done exactly what needs to be done 1) removing the symlink to `functorch/test` and 2) moving all the tests to `test/functorch`.